CMakeLists.txt

```diff
@@ -292,6 +292,10 @@ if (NOT USE_SIMD STREQUAL "")
             else ()
                 add_definitions ("-m${feature}")
             endif ()
+            if (feature STREQUAL "fma" AND CMAKE_COMPILER_IS_CLANG)
+                # Clang needs an explicit flag to make fma ops
+                add_definitions ("-ffp-contract=fast")
```

Comment:
None of this takes effect unless you build with the -mfma (triggered by fma being one of the options in USE_SIMD=...).

Lerp -- oy, isn't there ever any good news where new hardware makes things easier for programmers?